lib/facter.rb

```diff
@@ -168,6 +168,15 @@ def reset
       nil
     end
 
+    # Loads all facts
+    #
+    # @return [nil]
```

Comment:
I did a test and it returns an empty array. I think you need to add a nil at the end of the method.